Libraries/LibWeb/CSS/CalculatedOr.h

```diff
@@ -18,7 +18,7 @@
 
 namespace Web::CSS {
 
-template<typename T>
+template<typename Self, typename T>
```

Comment:
I think adding a `using ResultType = ...;` to the implementors à la `ErrorOr` and hooking into that would be even nicer